docs/source/basic_tutorials/non_core_models.md

```diff
@@ -0,0 +1,21 @@
+# Non-core Model Serving
+
+TGI supports various LLM architectures (see full list [here](https://github.com/huggingface/text-generation-inference#optimized-architectures)). If you wish to serve a model that is not one of the supported models, TGI will fallback to transformers implementation of that model. They can be loaded by:
```

Comment:
We should be linking to https://huggingface.co/docs/text-generation-inference/supported_models I think